### ISSUE TITLE
Create forks don't work

### ISSUE THREAD

**Issue by EvieTheOwl** [talrey/CreateDeco#105]
Unfortunately most peeps aren't using the main version of Create atm due to the Sodium issue that's been ongoing for a while now. Removing the hard dependency on build 1161 could be handy.

<img width="993" alt="Screenshot 2023-12-15 at 10 51 14" src="https://github.com/talrey/CreateDeco/assets/46009144/3edd7ce7-de0f-4a1a-a9a9-3db64d027a85">

**Comment by EvieTheOwl** [comment]
The 2 main forks atm are the Create DM Earth Edition i'm using atm and the Create sodium fix mentioned in #103

**Comment by Bugord** [comment]
Temporary fix
![image](https://github.com/talrey/CreateDeco/assets/22936782/4634a4ce-a236-40b3-9be4-948eb41e7201)

**Comment by Poopooracoocoo** [comment]
Don't necessarily need to modify the jar. 

<details>
<summary>Can use a fabric dependency override instead </summary>

Add a file in your config folder with the name fabric_loader_dependencies.json and the following contents:

```
{
  "version": 1,
  "overrides": {
    "createdeco": {
      "-depends": {
        "create": ""
      }
    }
  }
}

```

</details>

<details>
<summary>Or if you're using Quilt</summary>

You can create a [Quilt Dependency Override](https://github.com/QuiltMC/quilt-loader/wiki/Dependency-Overrides) to remove the requirement. 

Go to your `.minecraft` folder and then go to your `config` folder. Then create `quilt-loader-overrides.json` with the following

````
{
  "schema_version": 1,
  "overrides": [
      {
          "path": "<mods>/createdeco-2.0.0-fabric-1.20.1.jar",
          "depends": {
              "remove": {
                  "id": "create",
                  "versions": "*"
              }
          }
      }
  ]
}
````

</details>

**Comment by EvieTheOwl** [comment]
> Don't necessarily need to modify the jar.
> 
> Can use a fabric dependency override instead
> Add a file in your config folder with the name fabric_loader_dependencies.json and the following contents:
> 
> ```
> {
>   "version": 1,
>   "overrides": {
>     "createdeco": {
>       "-depends": {
>         "create": ""
>       }
>     }
>   }
> }
> ```

This is great for fabric but doesn't work for Quilt. You have to follow this process: https://github.com/QuiltMC/quilt-loader/wiki/Dependency-Overrides which I don't think is as simple. i figured it out if anyone needs help `@evieowlette (discord)` but yeah

**Comment by EvieTheOwl** [comment]
Just a heads up there is yet another fork and i've switched to this because it has better compatibility: https://www.curseforge.com/minecraft/mc-mods/prominent-create

3 Sodium forks chillin in the wild, which one shall we choose which one is desired :3

**Comment by solonovamax** [comment]
In what way does it have better compat?

**Comment by drewcassidy** [comment]
Was this resolved with 2.0.1?

**Comment by AstralOrdana** [comment]
it was
